pkg/git/fetch.go

```diff
@@ -18,6 +18,10 @@ const (
 
 type Fetch struct{}
 
+func NewFetcher() *Fetch {
```

Comment:
I would have expected the name of the struct.

```suggestion
func NewFetch() *Fetch {
```